app/src/main/java/com/travelbetadisaster/travel_log/ui/bucketList/BucketListAdapter.kt

```diff
@@ -57,6 +57,8 @@ class BucketListAdapter(private val entryListener: OnEntryClickListener,
         var title: TextView
         var description: TextView
         var completed: ImageView = itemView.findViewById(R.id.check_mark)
+        //todo: find a checkmark resource
```

Comment:
I actually implemented some in a previous commit. They should be found under 
drawable/baseline_check_box_24.xml
drawable/baseline_check_box_outline_blank_24.xml